frontend/src/pages/ApartmentPage.tsx

```diff
@@ -60,18 +66,21 @@ const ApartmentPage = (): ReactElement => {
   useEffect(() => {
     get<ReviewWithId[]>(`/review/aptId/${aptId}`, {
       callback: setReviewData,
+      errorHandler: handlePageNotFound,
```

Comment:
The error handler should not be included in the `get<ReviewWithId[]>(/review/aptId/${aptId}` or  `get<Landlord>(/landlord/${apt?.landlordId}` useEffect since the 404 should only show if the `aptId` is not correct. If you test this with any route, e.g. `/apartment/10`, it goes to 404 when it shouldn't because 10 is a valid `aptId` in the `buildings` collection.